src/components/TopNavigation/TopNavigation.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import NavigationLinks from '../NavigationLinks/NavigationLinks'
+import './TopNavigation.scss'
+import Cart from '../cart/cart'
+
+const TopNavigation = () => {
+  const navLinks = [
+    {
```

Comment:
NavLinks here is static and unchangeable, define it outside of component
And for better structure add it to another file in same scope called `constant.js` and export it from there